packages/customer-account-ui-extensions-react/src/render.tsx

```diff
@@ -28,9 +31,15 @@ export function render<ExtensionPoint extends RenderExtensionPoint>(
   const extendCallback: RenderFunction<any> = (root, api) => {
     return new Promise<void>((resolve, reject) => {
       try {
-        remoteRender(render(api), root, () => {
-          resolve();
-        });
+        remoteRender(
+          <ExtensionApiContext.Provider value={api}>
+            {render(api as ApiForRenderExtension<ExtensionPoint>)}
+          </ExtensionApiContext.Provider>,
```

Comment:
That's really cool, I thought it would need to be on the host but apparently not 🎉 

You can ignore my other comment then! 👍